src/core/components/learners/ProfileCourses.vue

```diff
@@ -91,7 +91,7 @@ export default {
   },
   data () {
     return {
-      isVendorInterface: /\/ad\//.test(this.$router.currentRoute.path),
+      isVendorInterface: /\/ad\//.test(this.$router.currentRoute.value.path),
```

Comment:
Il y en a encore pas mal d'autre, il faudrait qu'on tranche entre `this.$router.currentRoute.value.path` et `this.$route.path`

Reply:
j'ai tout changé en  `this.$route.name` que je trouvais plus clair et plus concis :)